source/reference/minio-mc/mc-replicate-diff.rst

```diff
@@ -0,0 +1,116 @@
+.. _minio-mc-replicate-diff:
+
+=====================
+``mc replicate diff``
+=====================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+.. mc:: mc replicate diff
+
+Description
+-----------
+
+.. start-mc-replicate-diff-desc
+
+The :mc:`mc replicate diff` shows object versions that are not replicated, either generally for the bucket or to a specific remote bucket.
+
+.. end-mc-replicate-diff-desc
+
+Syntax
+------
+
+.. tab-set::
+
+   .. tab-item:: EXAMPLE
+
+      The following command shows unreplicated objects for the ``teamorange/projects`` prefix in the ``notes`` bucket for the ``myminio`` ALIAS:
+
+      .. code-block:: shell
+         :class: copyable
+
+         mc replicate diff myminio/notes/teamorange/projects
+
+   .. tab-item:: SYNTAX
+
+      The command has the following syntax:
+
+      .. code-block:: shell
+         :class: copyable
+
+         mc [GLOBALFLAGS] replicate diff   \
+                          [--arn "string"] \
+                          TARGET
+
+      .. include:: /includes/common-minio-mc.rst
+         :start-after: start-minio-syntax
+         :end-before: end-minio-syntax
+
+Parameters
+~~~~~~~~~~
+
+.. mc-cmd:: TARGET
+   :required:
+
+   The path to the alias, prefix, or object.
+
+.. mc-cmd:: arn
+   :optional:
+
+   The Amazon Resource Number (ARN) to use for the request.
+
+   When specified, the command returns the replication status of the ARN.
+   If not specified, the command returns the general replication status of the bucket.
+
+Global Flags
+~~~~~~~~~~~~
+
+.. include:: /includes/common-minio-mc.rst
+   :start-after: start-minio-mc-globals
+   :end-before: end-minio-mc-globals
+
+Examples
+--------
+
+View Unreplicated Versions of Objects at a Prefix
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Display unreplicated versions of objects for a prefix:
+
+.. code-block:: shell
+   :class: copyable
+
```

Comment:
Still a little confused here - without --arn , does this not just return the status?

Reply:
Or does this behavior only matter if you have multiple remotes for a bucket?

Reply:
Corrected throughout. I was misunderstanding what this output.
It's a list of `PUT` and `DELETE` actions that have not yet replicated.